assets/js/ga.js

```diff
@@ -0,0 +1,33 @@
+var _dntStatus = navigator.doNotTrack || navigator.msDoNotTrack;
+
+var fxMatch = navigator.userAgent.match(/Firefox\/(\d+)/);
+
+var ie10Match = navigator.userAgent.match(/MSIE 10/i);
+
+var w8Match = navigator.appVersion.match(/Windows NT 6.2/);
+
+if (fxMatch && Number(fxMatch[1]) < 32) {
```

Comment:
Hm, is there a reason that `var _dntStatus = navigator.doNotTrack || navigator.msDoNotTrack || "Unspecified"` is not good enough? Testing FF version numbers or even testing for a version of IE that was fully discontinued in 2016 seems like we're shimming too much. If the browser's been configured to lie in its `navigator.doNotTrack || navigator.msDoNotTrack` response, then the user clearly messed with their browser and we don't need special considerations?

Reply:
Lets drop the extra checks, and only look for `var _dntStatus = navigator.doNotTrack || navigator.msDoNotTrack || "Unspecified"` as @Pomax has suggested